src/Driver/PDO/SQLite/Driver.php

```diff
@@ -46,7 +46,7 @@ public function connect(array $params)
         }
 
         UserDefinedFunctions::register(
-            [$pdo, 'sqliteCreateFunction'],
+            call_user_func([$pdo, 'sqliteCreateFunction']),
```

Comment:
This is wrong. `register()` received a method as a callable, but you're actually calling the method now.

Reply:
As you can see, this change breaks the SQLite testsuite bigtime.

Reply:
Yes, you are right. I changed it back.